### PR TITLE
Download button enhanced

### PR DESCRIPTION
**Pull request by MAZHARMIK**
If you find it ok, then you can merge it for now . But still I have a problem that we need to discuss once we meet.
Thanks

### CODE REVIEW

File: tool.css [hunk 1]
```diff
@@ -7,9 +7,20 @@ body {
 	color:white;
 }
 
+.mik{
```
Comment (gbelwariar) [inline tool.css]: Really great work! Can you change 'mik' to something more project-specific, e.g - 'metasub-download'?

Comment (MAZHARMIK) [reply tool.css]: Yes sure. I was just testing it first so I put what randomly came in my mind like always "mik". LOL

Comment (gbelwariar) [reply tool.css]: :+1:

Comment (MAZHARMIK) [reply tool.css]: Done. check it

Comment (gbelwariar) [reply tool.css]: FYI, you can make subsequent changes in the same PR itself. I am deleting the old PRs as the new ones are merged.

Comment (MAZHARMIK) [reply tool.css]: Ok I got it. Thanks

**Pull request closed without merging**